pages/new-page.md

```diff
@@ -1,7 +1,7 @@
 ---
 permalink: /adding-a-new-page/
 layout: default
-title: Adding a new page
+title: Adding a New Page
```

Comment:
@awfrancisco actually changed this to lowercase per [18F content guidelines](https://pages.18f.gov/content-guide/capitalization/) in #8.